clients/src/main/resources/common/message/LeaderChangeMessage.json

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "type": "data",
+  "name": "LeaderChangeMessage",
```

Comment:
Not something we have to do in this patch, but it's a little annoying that the generated class still gets the Data suffix. This was only added to avoid naming collisions with the existing Request/Response classes, so it should be unnecessary when the type is "data." I will file an upstream issue to fix this.

Reply:
Cool, feel free to get a newbie ticket :)